backend/app/main.py

```diff
@@ -158,7 +159,9 @@ async def fetch_data_product(
     return JSONResponse(resp.json(), resp.status_code)
 
 
+router.include_router(consents_router)
```

Comment:
I got a bit confused at first why this line starts with `router.` and the other lines with `app.`, until I realized that the consent router is included under the `/api` path. Not sure that should we consider refactoring `router` -> `api_router` to make it clear? 

Also wondering that would it work to specify it as `app.include_router(consent_router, prefix="/api/consent")` or will it cause a conflict. Of course that would then change the path to all of the routes, which we might not want (even id it might work).